scripts/ci/github_actions/driver/setup_ci_dependencies.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -xe
+set +o pipefail
+
+install_ci_dependencies (){
+  python -m pip install --upgrade pip
+  echo docker-hub==2.2.0 > dev-requirements.txt
+  pip install -r dev-requirements.txt
+}
+
+get_driver_version (){
+  source /home/runner/.bash_profile
+  cd common
+  image_version=`yq eval .identity.version config.yaml`
+  cd -
+}
+
+install_ci_dependencies
+cat >>/home/runner/.bash_profile <<'EOL'
+yq() {
+  docker run --rm -i -v "${PWD}":/workdir mikefarah/yq "$@"
+}
+EOL
```

Comment:
`install_yq()`?
and shouldn't it be inside `install_ci_dependencies`?

Reply:
done

Reply:
done

Reply:
I still think we need an `install_yq` function, even if it's called from inside `install_ci_dependencies`

Reply:
done